### ISSUE TITLE
[Feature Request] Track runs in self deployed MLFlow server

### ISSUE THREAD

**Issue by ankitbko** [microsoft/promptflow#2210]
I would like to track the experiment runs on self deployed MLFlow server. Currently I don't find any ways to pass `mlflow_tracking_uri` to `PFClient` and have it track everything in mlflow server. This feature is centrally track the experiment runs when not using AML to run the flows.

**Comment by D-W-** [comment]
Hi @ankitbko , we don't have plans to support this in the near future. I will place it on the long-term roadmap.

**Comment by melmatlis** [comment]
Is there a mitigation in which we can track local PF runs alongside AML runs, compare metrics etc.?

**Comment by D-W-** [comment]
Hi @melmatlis , we are working on a feature to track local run in cloud, + @wangchao1230  for more details.

**Comment by wangchao1230** [comment]
Yes. We are designing the experience to track local PF runs in AML workspace. I will update here if we have something to share.
